src/tbb/CMakeLists.txt

```diff
@@ -113,6 +113,8 @@ target_link_libraries(tbb
 
 tbb_install_target(tbb)
 
+set(TBB_LIB_NAME tbb)
```

Comment:
Consider grouping all pc-related stuff in one place and more obvious naming, e.g. `TBB_PC_LIB_NAME` or even `_tbb_pc_lib_name` (and `_tbb_pc_filename`/`_tbb_pc_name` below)